pyproject.toml

```diff
@@ -52,8 +54,25 @@ norecursedirs = [
     "share",
     ".tox",
     ".git",
-    "docs"
+    "docs",
 ]
 
 [tool.mypy]
-exclude = ["venv"]
\ No newline at end of file
+exclude = [".venv", "build", "noxfile.py", "tests"]
+
+[tool.ruff]
```

Comment:
is there something more to configure for ruff here? maybe we want an explicit base `select` slightly different than the default?

Reply:
we didn't use isort, so I haven't added [I] as I normally do in other projects. I would leave the restricted selections to the specific nox sessions meant for the CI.

Reply:
ok